### ISSUE TITLE
Update Amenoma version, and update scanner Game Manipulation string

### ISSUE THREAD

**Issue by nguyentvan7** [frzyc/genshin-optimizer#1008]
We should reword it to be "automated input" as "game manipulation" sounds very hacky/cheaty and can be misleading

**Comment by frzyc** [comment]
Isn't that the point of the term? It's the primary thing people worry about when they use auto scanners

**Comment by nguyentvan7** [comment]
game manipulation sounds like memory hacks or something (perhaps I am looking at it too technically). Automated input is less negative while still explaining what it does in a more specific way
